pkg/kubelet/nodelease/controller.go

```diff
@@ -98,12 +98,10 @@ func (c *controller) sync() {
 		// If at some point other agents will also be frequently updating the Lease object, this
 		// can result in performance degradation, because we will end up with calling additional
 		// GET/PUT - at this point this whole "if" should be removed.
-		lease, err := c.leaseClient.Update(c.newLease(c.latestLease))
+		err := c.retryUpdateLease(c.newLease(c.latestLease))
```

Comment:
if err := c.retryUpdateLease(c.newLease(c.latestLease)); err == nil {
  return 
}

Reply:
In this case, the `err` would be undefined.

Reply:
For logging purposes below - I agree.

Reply:
ok, LTGM